ceph/rados/rados_scrub.py

```diff
@@ -0,0 +1,159 @@
+"""
+   This module contains the methods required for scrubbing.
+
+   1.To set the parameters for scrubbing initially required the
+     cluster time and day details.get_cluster_date method provides
+     the details
+
+   2.set_osd_configuration method  used to set the configuration
+     parameters on the cluster.
+
+   3.get_osd_configuration  method is used to get the configured parameters
+     on the cluster.
+
+     NOTE: With set_osd_configuration & get_osd_configuration methods can
+          use to set the get the any OSD configuration parameters.
+
+   4. get_pg_dump  method is used to get the pg dump details from the cluster
+
+   5. verify_scrub  method used for the verification of scheduled scrub
+      happened or not.
+"""
+
+import logging
+from collections import defaultdict
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+
+log = logging.getLogger(__name__)
+
+
+class RadosScrubber:
+    def __init__(self, node: CephAdmin):
+        """
+        initializes the env to run rados commands
+        Args:
+            node: CephAdmin object
+        """
+        self.node = node
+        self.ceph_cluster = node.cluster
+        self.rados_obj = RadosOrchestrator(node=node)
+
+    def run_container_cmd(self, cmd: str):
+        """
+        Runs the command on conatiner and returns
+        the output
+        Args:
+            cmd: Command that needs to be run on container
+
+        Returns : string  value
+        """
+
+        cmd = f"{cmd}"
+        out, err = self.node.shell([cmd])
+        return out
+
+    def get_cluster_date(self):
```

Comment:
This is a cluster specific operation... meaning that you would need the time being set in the cluster. The cluster deployment can be baremetal or container.

The current implementation would limit you to container specific deployment. 

If that this the intention, we may want to document it.

Reply:
Moved the get_cluster_date  method to the core_workflows.py